src/arduino/pressure_controller.h

```diff
@@ -163,8 +163,19 @@ class PressureController
      */
     void safeguards(uint16_t p_centiSec);
 
+    /*! This function implements safeguard for peak pressure
+     * \param p_centiSec Current progress in the respiratory cycle in hundredth of second
+     */
+    void safeguardPressionCrete(uint16_t p_centiSec);
+
+    /*! This function implements a first safeguard for peep pressure
+     * \param p_centiSec Current progress in the respiratory cycle in hundredth of second
+     */
     void safeguardHoldExpiration(uint16_t p_centiSec);
 
+    /*! This function implements a second safeguard for peep pressure
```

Comment:
Ça vaudrait le coup d'expliquer la différence avec le 1er safeguard for peep pressure, non ?

Reply:
Oui tout à fait.